MIMWebClient/Core/Events/Score.cs

```diff
@@ -64,7 +64,7 @@ public static void UpdateUiInventory(Player playerData)
 
 
                 context.Clients.Client(playerData.HubGuid)
-                    .updateInventory(playerData.Inventory.Where(x => x.location == Item.Item.ItemLocation.Inventory && x.type != Item.Item.ItemType.Gold));
+                    .updateInventory(ItemContainer.List(playerData.Inventory.Where(x => x.location == Item.Item.ItemLocation.Inventory && x.type != Item.Item.ItemType.Gold)));
```

Comment:
Pass true in the itemcontainer list so we get the article showing before the item.